### PR TITLE
[FIX] Pinning reactions doesn't work as expected in #accountability-station. #171

### LINKED ISSUE #171: [BUG REPORT] Pinning reactions doesn't work as expected in #accountability-station.
**Issue description.**
When pinning a message in #accountability-station (i.e. reacting with the 📌 emoji), Horace pins the message as expected, but first he adds a reaction of his own and then removes the reactions altogether, making it harder to unpin later.

**How to reproduce.**
1. Send a message in #accountability-station.
2. React with the 📌 emoji. 

**Expected behavior.**
I see two possibilities:
1. Horace doesn't react at all (except, of course, the message), so the reaction is inevitably cleared when the user unpins by un-reacting. The reactions could also be cleared after using `!unpin`.
2. Horace reacts, but clears the reactions only after the user unpins the message (either by un-reacting or via `!unpin`).

**Discord information.**
Not relevant.

**Smartphone.**
Not relevant.

**Extra context.**
None.

### PR DESCRIPTION
**Pull request by catalincodes**
Fixed the issue/bug from:

https://github.com/knights-of-academia/horace/projects/3#card-53700051

Issues were: 
- partials were missing from Client object creation (for messages and reaction)
- commands were using obsolete methods that don't exist anymore like `channel.fetchPinnedMessages()`. I replaced all such problems with the new style recommended by DiscordJS.

Make sure to move the card from Backlog to Review In Progress and let me know if there's anything you would like to me to fix, before merging it.

### CODE REVIEW

**Comment by ghost** [comment]
Can you run a test and make sure that the following still work with the new partials in place as well?

- Terms and Conditions
- Focused Raider
- Bookmarks

**Comment by catalincodes** [comment]
I have checked and all three features seems to work just fine. Feel free to come and check them on the test server.

File: eventActions/accountabilityActions.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class accountabilityActions {
 			const currentChannel = sentMessage.channel;
 
 			// Check if there are too many existing pins
-			currentChannel.fetchPinnedMessages().then(messages => {
```
Comment (debashisbiswas) [inline eventActions/accountabilityActions.js:28]: Good fix. This isn't related to your PR, but there might be other parts of the codebase using the old function that will need the same change. Just something to keep in mind

File: eventActions/accountabilityActions.js [hunk 2]
```diff
@@ -84,7 +84,8 @@ class accountabilityActions {
 			let hasPinnedMessage = false;
 
 			// Get the pinned messages within a channel
-			await currentChannel.fetchPinnedMessages().then(fetchedPins => {
+			//currentChannel.messages.fetchPinned()
```
Comment (debashisbiswas) [inline eventActions/accountabilityActions.js]: Did you leave this comment in on accident?

Comment (catalincodes) [reply eventActions/accountabilityActions.js]: Yes... removing it when I get home and pushing a commit to the PR

Comment (debashisbiswas) [reply eventActions/accountabilityActions.js]: No worries! Just wanted to confirm.

**Comment by catalincodes** [comment]
Are there any existing issues, right now, that need to be linked before we finalize the PR? @KO4JZT @debashisbiswas @ThatNerdSquared

**Review by debashisbiswas — APPROVED** [review]
Looking good 👍

**Pull request merged**